src/mame/drivers/ssv.cpp

```diff
@@ -1835,9 +1834,9 @@ static INPUT_PORTS_START( mslider )
 	PORT_DIPUNUSED_DIPLOC( 0x0080, 0x0080, "DSW1:8" ) /* Manual lists this dip as "Unused" */
 
 	PORT_MODIFY("DSW2") // IN1 - $210004
-	PORT_DIPNAME( 0x0001, 0x0001, DEF_STR( Flip_Screen ) )  PORT_DIPLOCATION( "DSW2:1" )
-	PORT_DIPSETTING(      0x0001, DEF_STR( Off ) )
-	PORT_DIPSETTING(      0x0000, DEF_STR( On ) )
+	PORT_DIPNAME( 0x0001, 0x0000, DEF_STR( Flip_Screen ) )  PORT_DIPLOCATION( "DSW2:1" )
+	PORT_DIPSETTING(      0x0001, DEF_STR( On ) ) // service mode calls this OFF
+	PORT_DIPSETTING(      0x0000, DEF_STR( Off ) ) // and this ON, TODO: check if it's an error in the video code, or a mistake in the game
```

Comment:
Is it just designed for a cabinet with an inverted monitor?  It wouldn't be the first time.